### ISSUE TITLE
Name&syntax: Build node

### ISSUE THREAD

**Issue by superstar54** [aiidateam/aiida-workgraph#48]
## Build node from others
One can build a node from an already existing AiiDA component: `calcfunction`,  `workfunction`, `calcjob`, `Workchain` with the `build_node` function.

```python
from aiida_worktree import build_node
AddNode = build_node("aiida.calculations.arithmetic.add.ArithmeticAddCalculation")

# or
from aiida.calculations.arithmetic.add import ArithmeticAddCalculation
AddNode = build_node(ArithmeticAddCalculation)
```

One can also build a node from a WorkTree
```python
wt1 = WorkTree()
wt1.nodes.new(add, name="add")
wt1.nodes.new(multiply, name="multiply", x=wt1.nodes["add"].outputs[0])
# create a node using the WorkTree
AddMultiplyNode = build_node(wt1)
```

is `build_node` a good name/syntax?

**Comment by superstar54** [comment]
From @mbercx:
> I think the syntax of build_node is rather cumbersome. Why can't I just import the PwCalculation and then do add it with wt1.nodes.new? It should be able to figure out the path and build the node on the fly, no?

**Comment by superstar54** [comment]
Hi @mbercx, Thanks for the great suggestion! Indeed, it would be very user-friendly if we could use the AiiDa component directly. This is implemented in #50.
